packages/grafana-ui/src/options/builder/legend.tsx

```diff
@@ -11,21 +11,33 @@ export function addLegendOptions<T extends OptionsWithLegend>(
   builder
     .addRadio({
       path: 'legend.displayMode',
-      name: 'Legend mode',
+      name: 'Mode',
       category: ['Legend'],
       description: '',
       defaultValue: LegendDisplayMode.List,
       settings: {
         options: [
           { value: LegendDisplayMode.List, label: 'List' },
           { value: LegendDisplayMode.Table, label: 'Table' },
-          { value: LegendDisplayMode.Hidden, label: 'Hidden' },
+        ],
+      },
+    })
+    .addRadio({
```

Comment:
Please use a boolean switch instead of radio button. Similar to how we handle boolean options in other panels. Here's Table panel for reference:

<img width="447" alt="image" src="https://user-images.githubusercontent.com/2376619/180225539-0964f3ef-c17a-4f1f-9d1c-a76b71065eea.png">

Reply:
done!